pandas/core/frame.py

```diff
@@ -1606,12 +1607,37 @@ def to_latex(self, buf=None, columns=None, col_space=None, header=True,
 
             .. versionadded:: 0.18.0
 
+        multicolumn : boolean, default True
+            default will be read from the config module
+            Use \multicolumn to enhance MultiIndex columns.
```

Comment:
Can you use a proper sentence (Capital, punctuation) for the first line of 'default ..' (the thing is, here this looks OK in plain text, but in the html docs, the line breaks are not preserved, so will look line one long sentence without '.' in the middle)

Also, would start with the actual explanation of the keyword, and put "Default will be ..." at the end.